kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/ResourceSelection.vue

```diff
@@ -123,14 +97,27 @@
         @searchterm="handleSearchTermChange"
       />
 
+      <div
+        v-if="showNumberOfQuestionsWarning"
+        class="shadow"
+        :style=" {
+          padding: '1em',
+          marginTop: '2em',
+          marginBottom: '2em',
+          backgroundColor: $themePalette.grey.v_100,
+        }"
+      >
+        {{ cannotSelectSomeTopicWarning$({ count: 500 }) }}
```

Comment:
I am not seeing any updates to this string, which presumably will now say you can't select folders with more than 500 exercises instead.

May also be good to update the message id to be more about number of questions?

Reply:
The [updated string](https://github.com/learningequality/kolibri/pull/12312/files/5a50ffe0973046aa4de4d5e383c03f3a2efdbf69#diff-0d91967dccc31f8215a1cfe44f40959b46c60237de896768b5dd3aa9d37a0796R213) is this: 

'You can only select entire folders when they have { count, number } questions or fewer.',

Is that what you mean? 

although this does make me realize I should probably be using the const.... _const_istently! and not just adding 500 in random places